src/games/strategy/triplea/ai/proAI/util/ProBattleUtils.java

```diff
@@ -378,8 +378,6 @@ public boolean territoryHasLocalNavalSuperiority(final Territory t, final Player
         + ", alliedSize=" + myUnits.size() + ", enemySize=" + enemyUnitsInSeaTerritories.size());
     // If I have naval attack/defense superiority then break
     return (defenseStrengthDifference < 50 && attackStrengthDifference > 50);
-    // return Math.min(100, Math.min(200 - 2 * defenseStrengthDifference, 2 * attackStrengthDifference)); // 100 is good
-    // while anything less
-    // needs more units
```

Comment:
Hmm, the conversation about dependencies was misguided. The question is why are we failing to collaborate effectively and are duplicating each others work? This comment removal was already done in another PR. We're spending time on the exact same efforts, only getting the benefit once (and worst, the cost of resolving merge conflicts). It's not efficient or a good use of our combined efforts to be duplicating work.